ts/test/test-heap-profiler.ts

```diff
@@ -15,53 +15,116 @@
  */
 
 import * as delay from 'delay';
+import {describe} from 'mocha';
```

Comment:
Why was this needed? We were using `describe` before just fine. Note that we are not importing any of the other globals added by `mocha` (e.g. `beforeEach`).

Reply:
removed -- this isn't needed